js/ui/cardCounter.js

```diff
@@ -0,0 +1,12 @@
+export const generateCounter = (appState) => {
```

Comment:
**praise:** Good job on passing state here, as argument - but it will be enough if you just pass `cardsAmount` instead of whole state. You don't need whole state